src/AltinnCore/Common/Services/Implementation/PDFSI.cs

```diff
@@ -128,5 +135,23 @@ private async Task<DataElement> StorePDF(Stream pdfStream, Instance instance)
                     content);
             }
         }
+
+        private static string GetUTF8String(byte[] data)
+        {
+            if (data == null || data.Length < 1)
```

Comment:
Instead of < 1, what do you think of this: !data.Any() ?

Reply:
Fixed